articles/active-directory/develop/developer-guide-conditional-access-authentication-context.md

```diff
@@ -216,6 +216,47 @@ Do not use auth context where the app itself is going to be a target of Conditio
 - [Use the Conditional Access auth context to perform step-up authentication for high-privilege operations in a web app](https://github.com/Azure-Samples/ms-identity-dotnetcore-ca-auth-context-app/blob/main/README.md)
 - [Use the Conditional Access auth context to perform step-up authentication for high-privilege operations in a web API](https://github.com/Azure-Samples/ms-identity-ca-auth-context/blob/main/README.md)
 
+## Authentication Context [ACRs] in Conditional Access expected behavior
+
+### Explicit Auth Context Satisfaction in Requests
+A client can explicitly ask for a token with an Auth Context (ACR) through the claims in the request's body. If an ACR was asked, Conditional Access will allow issuing the token with the asked ACR if all challenges were completed.
```

Comment:
I'd be explicit and consistently use ACRS instead of ACR. I'd also use "requested" instead of "asked".

Reply:
I believe section headers should be sentenced cased. I.E."Authentication context [ACRS] in Conditional Access expected behavior". I'd apply consistently across section headers.

Reply:
@bakerCaleb can you please also review the different sections / headers and if you would rather combine a few logically to make it clearer?